Update FAQs (#6026)
circleci/circleci-docs: jekyll/_cci2/faq.md

## Migration
{: #migration }

Visit the [Migration Introduction]({{ site.baseurl }}//2.0/migration-intro/) doc which links out to migration guides for various platforms.

### How do I migrate from Jenkins to CircleCI?
{: #how-do-i-migrate-from-jenkins-to-circleci }
{:.no_toc}

Refer to [Migrating From Jenkins]({{ site.baseurl }}/2.0/migrating-from-jenkins/) for conceptual differences between Jenkins and CircleCI.

### Can I use CircleCI without creating base images?
{: #can-i-use-circleci-without-creating-base-images }
{:.no_toc}

> Yes, CircleCI provide a selection of "convenience images" for use with the Docker executor. For a full list, along with usage instructions visit the [CircleCI Developer Hub](https://circleci.com/developer/images) and the [CircleCI Images guide]({{site.baseurl}}/2.0/circleci-images/).

Reviewer: ```suggestion
Yes, CircleCI provides a selection of "convenience images" for use with the Docker executor. For a full list, along with usage instructions, visit the [CircleCI Developer Hub](https://circleci.com/developer/images) and the [CircleCI Images guide]({{site.baseurl}}/2.0/circleci-images/).
```

## Troubleshooting
{: #troubleshooting }

### Why aren't my jobs running when I push commits?
{: #why-arent-my-jobs-running-when-i-push-commits }
{:.no_toc}
In the CircleCI application, check the individual job and workflow views for error messages. More often than not, the error is because of formatting errors in your `config.yml` file.

See [Writing YAML]({{ site.baseurl }}/2.0/writing-yaml/) for more details.

After checking your `config.yml` for formatting errors, search for your issue in the [CircleCI support center](https://support.circleci.com/hc/en-us).

### Why is my job queued?
{: #why-is-my-job-queued }
{:.no_toc}
A job might end up being **queued** because of a concurrency limit being imposed due to the plan you or your organisation are on. If your jobs are queuing often, you can consider [upgrading your plan](https://circleci.com/pricing/).

### Why are my builds queuing even though I'm on the Performance plan?
{: #why-are-my-builds-queuing-even-though-im-on-performance-plan }
{:.no_toc}

> In order to keep the system stable for all CircleCI customers, we implement different soft concurrency limits on each of the [resource classes](https://circleci.com/docs/2.0/configuration-reference/#resource_class). If you are experiencing queuing on your jobs, it's possible you are hitting these limits. Please [contact CircleCI support](https://support.circleci.com/hc/en-us/requests/new) to request raises on these limits.

Reviewer: ```suggestion
In order to keep the system stable for all CircleCI customers, we implement different soft concurrency limits on each of the [resource classes](https://circleci.com/docs/2.0/configuration-reference/#resource_class). If you are experiencing queuing on your jobs, it is possible you are hitting these limits. Please [contact CircleCI support](https://support.circleci.com/hc/en-us/requests/new) to request raises on these limits.
```

- **Server** - You install and manage CircleCI, through a service like AWS or GCP. Server installations are behind a firewall that your team sets up and maintains according to your datacenter policy. You have full administrative control for complete customization and management of upgrades as new versions are released.

## Troubleshooting
{: #troubleshooting }

### Why aren't my jobs running when I push commits?
{: #why-arent-my-jobs-running-when-i-push-commits }
{:.no_toc}
In the CircleCI application, check the individual job and workflow views for error messages. More often than not, the error is because of formatting errors in your `config.yml` file.

See [Writing YAML]({{ site.baseurl }}/2.0/writing-yaml/) for more details.

After checking your `config.yml` for formatting errors, search for your issue in the [CircleCI support center](https://support.circleci.com/hc/en-us).

### Why is my job queued?
{: #why-is-my-job-queued }
{:.no_toc}
A job might end up being **queued** because of a concurrency limit being imposed due to the plan you or your organisation are on. If your jobs are queuing often, you can consider [upgrading your plan](https://circleci.com/pricing/).

> **run queue** forms when CircleCI experiences high demand. Customer builds are placed in a run queue and processed as machines become available.  Time spent in a **run queue** is unavoidable (though CircleCI 

Author: ```suggestion
```
